js/webpack_get_defaults.js

```diff
@@ -80,11 +81,14 @@ module.exports = function (basePath) {
     })
   }
   if (config.build.type === enums.buildTypes.APPLICATION) {
-    plugins.push(new HtmlPlugin({
-      inject: true,
-      filename: 'index.html',
-      template: path.join(config.build.src_dir, 'index.html'),
-    }))
+    const indexHtmlPath = path.join(config.build.src_dir, 'index.html')
+    if (fs.existsSync(indexHtmlPath)) {
```

Comment:
Je suis en congés mais juste my 2 cents, selon moi c'est les 'libs' en
question (smart table etc) qu'il faut reprendre pour ne PAS inclure
l'univers (genre font-awesome) pour régler le souci.

Cassets pourrait inclure du sass pour que les apps l'utilisent au build
time mais ce qui est considéré comme de la 'library' et qui sera donc
précompilé SANS ses dépendances ne doit pas générer un fichier CSS de 300kb
juste pour une classe de styling dans smart table. Ça me semble raisonnable.
On May 27, 2016 15:10, "Raphaël Boucher" <notifications@github.com> wrote:

> In js/webpack_get_defaults.js
> <https://github.com/Polyconseil/systematic/pull/14#discussion_r64901765>:
>
> > @@ -80,11 +81,14 @@ module.exports = function (basePath) {
> >      })
> >    }
> >    if (config.build.type === enums.buildTypes.APPLICATION) {
> > -    plugins.push(new HtmlPlugin({
> > -      inject: true,
> > -      filename: 'index.html',
> > -      template: path.join(config.build.src_dir, 'index.html'),
> > -    }))
> > +    const indexHtmlPath = path.join(config.build.src_dir, 'index.html')
> > +    if (fs.existsSync(indexHtmlPath)) {
>
> Ah oui, en effet!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Polyconseil/systematic/pull/14/files/1611e941a979d7841ecae013a676766adffda650..eac4fa7909178bf2ef3eab644f9b2d265b0807c8#r64901765>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFaKqosfb-zF9bIr9-18ri-0gtDWRYPks5qFu1bgaJpZM4IntYR>
> .
>